src/main/java/com/syncstream_dev/syncstream/model/Genre.java

```diff
@@ -0,0 +1,11 @@
+package com.syncstream_dev.syncstream.model;
+
+public enum Genre {
```

Comment:
Alphabetical order.

Reply:
09d9f9147b0cb8cd05efc45bd22ad5a9b7f50ba1 addresses this